test/capi/capiPicture.cpp

```diff
@@ -66,6 +66,12 @@ TEST_CASE("Load Svg Data in Picture", "[capiPicture]")
     REQUIRE(w == Approx(1000).epsilon(0.0000001));
     REQUIRE(h == Approx(1000).epsilon(0.0000001));
 
+    //Verify Position
+    float x, y;
+    REQUIRE(tvg_picture_get_viewbox(picture, &x, &y, &w, &h) == TVG_RESULT_SUCCESS);
+    REQUIRE(x == Approx(0).epsilon(0.0000001));
+    REQUIRE(y == Approx(0).epsilon(0.0000001));
```

Comment:
btw, why do you only check x/y, not w/h?

Reply:
Becouse it was check with with tvg_picture_get_size in line 65, so I didn't if I had to check it again

Reply:
Unit test is to verify the apis work correctly or not.
In point of API perspective view, tvg_picture_get_size() and tvg_picture_get_viewbox() have no relationship with each others.
That means it needs test possible scenarios for tvg_picture_get_viewbox() return values.

Thus, please check all return values.

Reply:
Ok, I wll check it